.github/workflows/demo.yml

```diff
@@ -51,8 +51,25 @@ jobs:
                 "jws": "eyJhbGciOiJFZERTQSIsImI2NCI6ZmFsc2UsImNyaXQiOlsiYjY0Il19..HZtoLHUCGXalQH8VPClh0TcsQeNKS5i9KWLyASTQYfPIUPDMnLnjgjPJ5TVCn7S4CV7i45aTsUWkfs6cBNntBQ"
               }
             }
-          
-  
-
-    
-    
\ No newline at end of file
+  demo-get-credentials:
+    runs-on: ubuntu-latest
+    name: Get Credentials
+    steps:
+      - name: Get Access Token
+        uses: transmute-industries/verifiable-data-platform-github-action@v0.0.4
+        id: getAccessToken
+        with:
+          operation-id: getAccessToken
+          api-base-url: ${{ secrets.API_BASE_URL }}
+          did: ${{ secrets.ORGANIZATION_DID_WEB }}
+          token-endpoint: ${{ secrets.TOKEN_ENDPOINT }}
+          token-audience: ${{ secrets.TOKEN_AUDIENCE }}
+          client-id: ${{ secrets.CLIENT_ID }}
+          client-secret: ${{ secrets.CLIENT_SECRET }}
+      - name: Gets Credentials
+        uses: transmute-industries/verifiable-data-platform-github-action@v0.0.4
+        id: getCredentials
+        with:
+          operation-id: getCredentials
+      - name: Output
+        run: echo "${{ env.verifiable_data_platform_api_response }}"
```

Comment:
This command will echo this organization's credentials to a public repo's action runs... it is a bit questionable, but perhaps there is no better way to test.